### ISSUE TITLE
using fulton harness on advent VIP results in mission failure

### ISSUE THREAD

**Issue by adamzl** [MalucoMarinero/XCOM2GuerillaWarfare#41]
i did not knock out the advent VIP, the mission was marked failure and no rewards were gotten including the partial rewards for killing the vip.

next time the mission comes up i'll knock out out the vip then use fulton and confirm that works.

**Comment by MalucoMarinero** [comment]
yeah. I'll have to adjust the mission logic I think there. It won't listen for evac of the VIP until after you've done knockout I think, but I have control of the logic so it shouldn't be a big deal to update.
